kubernetes-go-guestbook/README.md

```diff
@@ -0,0 +1,11 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/kubernetes-go-guestbook/simple)
+
+# Simple Guestbook App
+
+A port of the standard [Kubernetes Guestbook](https://kubernetes.io/docs/tutorials/stateless-application/guestbook/)
+to Pulumi. This example shows you how to build and deploy a simple, multi-tier web application using Kubernetes and
+Docker, and consists of three components:
+
+* A single-instance Redis master to store guestbook entries
+* Multiple replicated Redis instances to serve reads
+* Multiple web frontend instances
```

Comment:
Note it would be *great* to add the "component based" one as well.  That helps highlight the value of using a real language.  That said, the component authoring experience is a little heavy weight as well - but the end user code should be massively simplified.  (similar to C# where it goes from 274 lines to 50 lines)